example_simulation/example.conf

```diff
@@ -0,0 +1,58 @@
+[conf_1]
+material = [gold_fcc]
+# Arg: Mandatory
+# Type: List or string
+# Discrp: Materials to be simulated. If using custom materials, the names shall
+#         correspond to atoms object(s) in the python file(s). If using materials project,
+#         use the prefix "mp_" followed by the formula to be searched for.
+materials_path = /path/to/example_simulation/[materials] 
+# Arg:    Optional
+# Type:   String
+# Discrp: The path where the materials are stored. Note: change the
+#         '/path/to/' to the selected path directory. Otherwise the 'startup' command 
+#         won't work. [materials] can be either a folder with python files or a python file.
+workspace_path = /path/to/example_workspace 
+# Arg:    Mandatory
+# Type:   String
+# Discrp: The selected path of the workspace. Note: if no example_workspace exist,
+#         a example_workspace will be created.
+ensemble = [NVT]
+# Arg:    Mandatory
+# Type:   String or list
+# Discrp: The ensemble/ensembles for the selected materials. If single then string, if
+#         many then a list.
+potential = [openkim]
+# Arg:    Mandatory
+# Type:   String or list
+# Discrp: Type of potential to be used on the materials. 
+kim-model = EAM_Dynamo_Ackland_1987_Au__MO_754413982908_00
+# Arg:    Mandatory if potential is openkim, otherwise optional.
+# Type:   String
+# Discrp: The string is selected from openkim. This string is generated by selecting
+#         the specific material in the periodic table from: https://openkim.org/
+initial-temperature = 40 
+# Arg:    Mandatory
+# Type:   Float
+# Discrp: The intial temprature of the simulation.

```

Comment:
initial* temperature*, also specify that it's in kelvin.

Reply:
done.